<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>selection_tasks_calender_window</class>
 <widget class="QWidget" name="selection_tasks_calender_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1167</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="tasks_pushbutton">
   <property name="geometry">
    <rect>
     <x>311</x>
     <y>232</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="calender_pushbutton">
   <property name="geometry">
    <rect>
     <x>509</x>
     <y>232</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="choosing_label">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>170</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>tasks_pushbutton</sender>
   <signal>clicked()</signal>
   <receiver>selection_tasks_calender_window</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>calender_pushbutton</sender>
   <signal>clicked()</signal>
   <receiver>selection_tasks_calender_window</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>609</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>794</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
